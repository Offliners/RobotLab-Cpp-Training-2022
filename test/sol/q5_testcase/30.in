(3+1*7+20/15+19*9*8)=/((4/20+5)*2/12+20*18)=*12+20%4=-17*18-6*20-12*6=
