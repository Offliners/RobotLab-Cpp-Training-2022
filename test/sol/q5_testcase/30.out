1379
1382
17966
35434
